src/www/scripts/app.js

```diff
@@ -595,7 +595,16 @@ module.exports = (function() {
             let remoteResult = await getRemoteConfig();
 
             if (shouldDownloadFn(remoteResult)) {
-                await synchronizePackage(sourceUri, destinationUri);
+                try {
+                    let localResult = await getLocalConfig();
+
+                    if (remoteResult.cachebust !== localResult.cachebust) {
+                        await synchronizePackage(sourceUri, destinationUri);
+                    }
+                } catch (e) {
+                    await synchronizePackage(sourceUri, destinationUri);
```

Comment:
This is the same as line `:602`. I'm assuming that `synchronizePackage` is not the possible cause of an exception? If not, then we can just log the exception, and then have the `synchronizePackage` call below the `try-catch`.

Reply:
The key element here is the if statement, which states that we only synchronize the package if:
- we fetched the local config;
- we fetched the remote config; and
- the cachebusts for the two are different

In case we fail to fetch the local config, we will always synchronize.